x-pack/filebeat/module/googlecloud/firewall/_meta/fields.yml

```diff
@@ -0,0 +1,138 @@
+- name: firewall
+  type: group
+  description: >
+    Fields for Google Cloud Firewall Rules logs.
+  fields:
+  - name: rule_details
+    type: group
+    description: >
+      Description of the firewall rule that matched this connection.
+    fields:
+      - name: priority
+        type: long
+        description: The priority for the firewall rule.
+      - name: action
+        type: keyword
+        description: Action that the rule performs on match.
+      - name: direction
+        type: keyword
+        description: Direction of traffic that matches this rule.
+      - name: reference
+        type: keyword
+        description: Reference to the firewall rule.
+      - name: source_range
+        type: keyword
+        description: List of source ranges that the firewall rule applies to.
+      - name: destination_range
+        type: keyword
+        description: List of destination ranges that the firewall applies to.
+      - name: source_tag
+        type: keyword
+        description: >
+          List of all the source tags that the firewall rule applies to.
+      - name: target_tag
+        type: keyword
+        description: >
+          List of all the target tags that the firewall rule applies to.
+      - name: ip_port_info
+        type: array
+        description: >
+          List of ip protocols and applicable port ranges for rules.
+      - name: source_service_account
+        type: keyword
+        description: >
+          List of all the source service accounts that the firewall rule applies to.
+      - name: target_service_account
+        type: keyword
+        description: >
+          List of all the target service accounts that the firewall rule applies to.
+
+  - name: destination.instance
```

Comment:
This group and some other below are duplicated from vpcflows. I think it's worth to move them to `googlecloud.destination.instance` etc. but I'm unsure about backwards-compatibility. Is setting the previous `googlecloud.vpcflow.xxx` field as an alias enough and safe?

Reply:
I think moving them up to the module level would be good. The vpcflow fileset is still in beta so we should be able to move those fields without adding the alias.